### ISSUE TITLE
[v-checkbox] missing density check

### ISSUE THREAD

**Issue by Flamenco** [vuetifyjs/eslint-plugin-vuetify#45]
Old syntax is not flagged.

old
```
 <v-checkbox  dense/>
```
new
```
 <v-checkbox  density="compact"/>
```
